howto/container/start.md

```diff
@@ -0,0 +1,17 @@
+When a container is either initialised with the `amc init` (see [Create a container](https://anbox-cloud.io/docs/howto/container/launch)) command or stopped with `amc stop` command (See [Stop a container](TBD)), you must start it explicitly with the `amc start` command:
+
+    amc start <container id>
+
+`<container id>` is the ID of the container that you want to start.
+
+By default, the `amc start` command waits 5 minutes for a container to run before the operation times out. Before starting a container, you can specify a custom wait time with the `--timeout` option.
```

Comment:
```suggestion
By default, the `amc start` command waits 5 minutes for a container to run before the operation times out. When starting a container, you can specify a custom wait time with the `--timeout` option.
```
With "before", it sounds like the command will first wait for 10 minutes and then start the container.